dotcom-rendering/src/web/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -195,15 +198,19 @@ const columnStyle = css`
 export const Column = ({
 	column,
 	index,
-	isLastColumn,
+	showLineBelow,
 }: {
-	column: PillarType;
+	column: PillarType | EditionLinkType;
 	index: number;
-	isLastColumn: boolean;
+	showLineBelow: boolean;
 }) => {
 	// As the elements are dynamic we need to specify the IDs here
-	const columnInputId = `${column.title}-checkbox-input`;
-	const collapseColumnInputId = `${column.title}-button`;
+	//Replace whitespace with hyphen https://stackoverflow.com/questions/3794919/replace-all-spaces-in-a-string-with/3795147#3795147
+	const columnInputId = `${column.title}-checkbox-input`.split(' ').join('-');
+	const collapseColumnInputId = `${column.title}-button`.split(' ').join('-');
+	const ariaControls = `${column.title.toLowerCase()}Links`
+		.split(' ')
+		.join('-');
```

Comment:
```suggestion
	const columnId = column.title.toLowerCase().split(' ').join('-')
	const columnInputId = `${columnId}-checkbox-input`;
	const collapseColumnInputId = `${columnId}-button`;
	const ariaControls = `${columnId}Links`;
```

To me this reads a little bit better? It does mean they're all lower case, but I wasn't sure if that would be a problem?